src/monocle_apptrace/wrap_common.py

```diff
@@ -257,21 +291,36 @@ def update_workflow_type(to_wrap, span: Span):
         if(package_name is not None and package in package_name):
             span.set_attribute(WORKFLOW_TYPE_KEY, workflow_type)
 
-def update_span_with_context_input(to_wrap, wrapped_args ,span: Span):
+def update_span_with_context_input(to_wrap, wrapped_args, instance ,span: Span):
     package_name: str = to_wrap.get('package')
     if "langchain_core.retrievers" in package_name:
         input_arg_text = wrapped_args[0]
         span.add_event(CONTEXT_INPUT_KEY, {QUERY:input_arg_text})
```

Comment:
Can you remove the duplication of the following line
span.add_event(CONTEXT_INPUT_KEY, {QUERY:input_arg_text})

Reply:
done